lib/theme-and-appearance.zsh

```diff
@@ -6,11 +6,11 @@ export LSCOLORS="Gxfxcxdxbxegedabagacad"
 
 if [[ "$DISABLE_LS_COLORS" != "true" ]]; then
   # Find the option for using colors in ls, depending on the version
-  if [[ "$(uname -s)" == "NetBSD" ]]; then
+  if [[ "$OSTYPE" == netbsd* ]]; then
```

Comment:
[Zsh `OSTYPE` comes from autoconf's `host_os`](https://github.com/zsh-users/zsh/blob/8f2ce89a0c80f6a0ecdc7c1189d6a94f620882f4/configure.ac#L46), and autoconf uses `host_os` to specify the bsds in [this way](http://git.savannah.gnu.org/gitweb/?p=autoconf.git;a=blob;f=lib/autoconf/functions.m4;h=66abe2946d3617e5d91686ee9c164f004c5850a0;hb=2b4844a760d0046b98d93256232d4774c23120be#l914). The BSD OSTYPEs here should be correct.